cmd/docker.go

```diff
@@ -86,9 +87,18 @@ func dockerHoverBuild(targetOS string, packagingTask packaging.Task, buildFlags
 		// intended to be abused and may disappear at any time.
 		dockerArgs = append(dockerArgs, "--env", "HOVER_IN_DOCKER_BUILD_VMARGS="+strings.Join(vmArguments, ","))
 	}
-	// TODO: Use hover container of version of the current running hover.
-	// 		Use debug package to obtain Module info which contains the version.
-	dockerImage := "goflutter/hover:latest"
+
+	buildInfo, ok := debug.ReadBuildInfo()
+	if !ok {
+		log.Errorf("Cannot obtain version information from hover build. To resolve this, please go-get hover using Go 1.13 or newer.")
+		os.Exit(1)
+	}
+	log.Infof("My buildInfo version is: %v", buildInfo.Main.Version)
+	version := buildInfo.Main.Version
+	if version == "(devel)" {
+		version = "latest"
```

Comment:
So what happens here is that if you `go install .`, then it'll use latest.

I use this command to test a local development build over hover with --docker functionality:
```bash
go install . && DOCKER_BUILDKIT=1 docker build . -t goflutter/hover:latest
```

Reply:
What is `DOCKER_BUILDKIT=1` for?

Reply:
It enables docker's buildkit based building of images, which is useful for us because of its parallel processing of multi-stage docker files.